src/app/models/base.py

```diff
@@ -0,0 +1,3 @@
+from sqlalchemy.orm import declarative_base
+
+Base = declarative_base
```

Comment:
`declarative_base` is a method creating class. 
According to [documentation](https://docs.sqlalchemy.org/en/13/orm/extensions/declarative/basic_use.html) in this case it should be:
`Base = declarative_base()`

Reply:
fixed